src/main/java/org/jitsi/videobridge/IceUdpTransportManager.java

```diff
@@ -199,8 +178,8 @@ public IceUdpTransportManager(Conference conference,
         conference.appendDiagnosticInformation(diagnosticContext);
         diagnosticContext.put("transport", hashCode());
 
-        iceAgent = createIceAgent(controlling, iceStreamName, true /* rtcpmux */);
-        iceStream = iceAgent.getStream(iceStreamName);
+        iceAgent = createIceAgent(controlling, DEFAULT_ICE_STREAM_NAME);
```

Comment:
the thing i didn't like about using the same name for all of them is it makes debugging a bit harder.  do you not think so?